controllers/productController.js

```diff
@@ -83,8 +85,59 @@ const getProduct = async (req, res) => {
   }
 };
 
+const deleteP = async (req, res) => {
+  const del_id = req.params.id;
+
```

Comment:
Lets destructure id instead of accessing req.params.id so it should look like :
const {id}=req.params

Reply:
done